app/views/directives/deploy-image.html

```diff
@@ -62,137 +62,139 @@
     <div ng-if="!loading" class="h2">Select an image stream tag or enter an image name.</div>
     <div ng-if="loading" class="h2">Loading image metadata for <span class="word-break">{{imageName | stripSHA}}</span>...</div>
   </div>
-  <div class="row mar-bottom-md" ng-if-start="!loading && import.image">
-    <div class="col-sm-12 mar-top-lg mar-bottom-lg">
-      <div class="separator"></div>
-    </div>
-    <div class="col-sm-2 hidden-xs text-right h2">
-      <span class="fa fa-cube text-muted" style="font-size: 100px;" aria-hidden="true"></span>
-    </div>
-    <div class="col-sm-10">
-      <div ng-if="runsAsRoot" class="alert alert-warning">
-        <span class="pficon pficon-warning-triangle-o" aria-hidden="true"></span>
-        Image <strong>{{import.name}}</strong> runs as the
-        <strong>root</strong> user which might not be permitted by your cluster
-        administrator.
+  <div ng-show="!loading && import.image">
```

Comment:
Changed this to an `ng-show` so that the "Deploy" button isn't enabled in the dialog before the form below is displayed.